### PR TITLE
replace youtube_transcript_api with youwhisper

### PR DESCRIPTION
**Pull request by tvpeter**
- replaced youtube_transcript_api with youwhisper 
- change default repo to bitcointranscripts for opening PRs
- fixes https://github.com/tvpeter/yt2btc/issues/2#issue-1516674946

### CODE REVIEW

File: yttbtc/cli.py [hunk 1]
```diff
@@ -7,22 +7,85 @@
...
+
+
+def convert_to_mp3(filename):
+    clip = VideoFileClip(filename)
+    clip.audio.write_audiofile(filename[:-4] + ".mp3")
+    clip.close()
+
+
+def AudiotoText(filename):
+    model = pywhisper.load_model("base")
```
Comment (adamjonas) [inline yttbtc/cli.py:37]: I'd maybe start with the tiny model and would certainly offer this as an option to toggle in the future.

File: yttbtc/cli.py [hunk 2]
```diff
@@ -47,37 +110,28 @@ def yt2btc(
     file_name_with_ext = file_name + '.md'
     outls = []
 
-    tx = YouTubeTranscriptApi.get_transcript(video_id)
-
     url = "https://www.youtube.com/watch?v=" + video_id
-
+    result = convert(url, 'base.en')
```
Comment (adamjonas) [inline yttbtc/cli.py:114]: 'base.en' should be a variable

File: yttbtc/cli.py [hunk 3]
```diff
@@ -47,37 +110,28 @@ def yt2btc(
...
         title = data['title']
-
+    
     meta_data = '---\n' \
                 f'title: {title} ' + '\n' \
-                'transcript_by: youtube_transcript_api\n' \
-                f'media: {url}\n' \
-                '---\n'
+                                     f'transcript_by: youtube_to_bitcoin_transcript_v_{__version__}\n' \
+                                     f'media: {url}\n' \
```
Comment (adamjonas) [inline yttbtc/cli.py:127]: spacing

File: yttbtc/cli.py [hunk 4]
```diff
@@ -47,37 +110,28 @@ def yt2btc(
...
-    tx = YouTubeTranscriptApi.get_transcript(video_id)
-
     url = "https://www.youtube.com/watch?v=" + video_id
-
+    result = convert(url, 'base.en')
+    
     query_string = urllib.parse.urlencode({"format": "json", "url": url})
     full_url = "https://www.youtube.com/oembed" + "?" + query_string
-
+
```
Comment (adamjonas) [inline yttbtc/cli.py:118]: spacing

File: yttbtc/cli.py [hunk 5]
```diff
@@ -47,37 +110,28 @@ def yt2btc(
...
     query_string = urllib.parse.urlencode({"format": "json", "url": url})
     full_url = "https://www.youtube.com/oembed" + "?" + query_string
-
+    
     with urllib.request.urlopen(full_url) as response:
         response_text = response.read()
         data = json.loads(response_text.decode())
         title = data['title']
-
+
```
Comment (adamjonas) [inline yttbtc/cli.py:123]: spacing

**Review by adamjonas — COMMENTED** [review]
Tested it out and it went surprisingly fast.

Failed in the push but seems unrelated to youwhisper swap.

File: yttbtc/cli.py (hunk 2 above)
Comment (tvpeter) [reply yttbtc/cli.py:114]: creating this as an issue

**Pull request merged**